src/renderer/component/fileActions/index.js

```diff
@@ -12,6 +12,7 @@ const select = (state, props) => ({
   /* availability check is disabled due to poor performance, TBD if it dies forever or requires daemon fix */
   costInfo: makeSelectCostInfoForUri(props.uri)(state),
   claimIsMine: makeSelectClaimIsMine(props.uri)(state),
+  claimId: props.claimId,
```

Comment:
Since this is already being passed in from another component, we don't need to specify it here. This is only for selecting props from state.